phys/module_sf_bep.F

```diff
@@ -18,7 +18,7 @@ MODULE module_sf_bep
 ! -----------------------------------------------------------------------
 
       integer nurbm           ! Maximum number of urban classes    
-      parameter (nurbm=3)
+      parameter (nurbm=11)
```

Comment:
I wonder about the memory implications for these increases in BEP and BEM. Are these used in combined multi-dimensional arrays?

Reply:
It is OK because these are not also spatially dimensioned.